unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/GltfContainer/GltfContainerCollidersHandler.cs

```diff
@@ -0,0 +1,180 @@
+            catch (Exception e) { Debug.LogException(e); }
+
+            return visibleMeshesColliders;
+        }
+
+        // Create visible meshes colliders if there are none created yet
+        // otherwise return the already created colliders
+        public IReadOnlyList<Collider> GetVisibleMeshesColliders()
+        {
+            try
+            {
+                if (visibleMeshesColliders.Count == 0)
+                    SetVisibleMeshesColliders(cachedRenderers, false);
+            }
+            catch (Exception e) { Debug.LogException(e); }
+
+            return visibleMeshesColliders;
+        }
+
+        public IReadOnlyList<Collider> GetInVisibleMeshesColliders()
+        {
+            return invisibleMeshesColliders;
+        }
+
+        private void SetVisibleMeshesColliders(IReadOnlyCollection<Renderer> renderers, bool onlyGltfStoredColliders)
+        {
+            const string POINTER_COLLIDER_NAME = "OnPointerEventCollider";
+
+            if (visibleMeshesColliders.Count > 0)
+                return;
+
+            foreach (Renderer renderer in renderers)
+            {
+                Transform rendererT = renderer.transform;
+
+                if (TryFindAlreadyCreatedCollider(rendererT, out Collider collider))
+                {
+                    collider.enabled = false;
+
+                    if (IsValidRendererForCollider(renderer)) { visibleMeshesColliders.Add(collider); }
+                    else { visibleMeshesInvalidColliders.Add(collider); }
+
+                    continue;
+                }
+
+                if (onlyGltfStoredColliders)
+                    continue;
+
+                MeshFilter meshFilter = renderer.GetComponent<MeshFilter>();
+                Mesh colliderMesh = meshFilter ? meshFilter.sharedMesh : null;
+
+                if (!colliderMesh)
+                    continue;
+
+                GameObject colliderGo = new GameObject(POINTER_COLLIDER_NAME);
+                MeshCollider newCollider = colliderGo.AddComponent<MeshCollider>();
+
+                newCollider.sharedMesh = colliderMesh;
+                newCollider.enabled = false;
+                colliderGo.transform.SetParent(renderer.transform);
+                colliderGo.transform.ResetLocalTRS();
+                visibleMeshesColliders.Add(newCollider);
+            }
+        }
+
+        // Compatibility layer for old GLTF importer and GLTFast
+        private static bool IsCollider(MeshFilter meshFilter)
+        {
+            const StringComparison IGNORE_CASE = StringComparison.CurrentCultureIgnoreCase;
+
+            return meshFilter.name.Contains("_collider", IGNORE_CASE)
+                   || meshFilter.transform.parent.name.Contains("_collider", IGNORE_CASE);
+        }
+
+        // Try to find if a collider is already created
+        private static bool TryFindAlreadyCreatedCollider(Transform transform, out Collider outCollider)
```

Comment:
if this is only for pointer event collider maybe it should be renamed to something like `TryToFindPointerEventCollider()`

Reply:
ended up removing that, since it was no needed, double checked and in sdk6 those mesh colliders are removed before moving the gltf back to the pool